rdmd.d

```diff
@@ -714,9 +725,207 @@ private string[string] getDependencies(string rootModule, string workDir,
         exit(depsExitCode);
     }
 
+    if (!hasDependencies)
+        std.file.remove(rootModule);
+
     return dryRun ? null : readDepsFile();
 }
 
+bool needsInspection(string file)
+{
+    import std.utf : byChar;
+
+    // an import statement can be at the beginning of a line or preceded by
+    // at least one whitespace
+    // a whitespace after `import` is mandatory
+    foreach (match; file.matchAll(ctRegex!(`(?<=^|\s)import\s+([^;]*);`, "ms")))
```

Comment:
`import` can also appear after these: `;{}`. There may be more. Same with `mixin` below.